### PR TITLE
Add osx and increase version

### LINKED ISSUE #12: Possible to create Mac version?
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

Hi, I was wondering if it might be possible to re-render this package and include a Mac version? I installed it via pip and that worked so I think it should be doable.

If there is interest, I can also try to create the PR but I just wanted to check first.

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #12 

Clean version of #17

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
Would be ready from my side @setu4993

**Comment by setu4993** [comment]
Thanks @BastianZim! I'll take a look at this later today.

Tagging @h-vetinari in case they wanted to take a look at this independently again.

**Comment by BastianZim** [comment]
Just fyi (Because I was adding another package and just saw this), sentencepiece has already been requested at the arm migration (https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1180)

File: recipe/build.sh [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
+    cmake \
+        -DCMAKE_INSTALL_PREFIX=$PREFIX \
+        -DCMAKE_INSTALL_LIBDIR=$PREFIX/lib \
+        -DCMAKE_AR="${AR}" \
+        -DSPM_ENABLE_TCMALLOC=OFF \
+        -S ..
+
+elif [[ "${target_platform}" == osx-* ]]; then
+
+    cmake .. -DSPM_ENABLE_SHARED=OFF -DCMAKE_INSTALL_PREFIX=../..
```
Comment (setu4993) [inline recipe/build.sh:20]: That's the configuration I was missing in my earlier attempts :).

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,39 +1,45 @@
...
-    - gperftools  # [unix]
 
 test:
   imports:
     - sentencepiece
   commands:
-    - spm_export_vocab --help  # [unix]
-    - spm_normalize --help  # [unix]
+    - pip check
+    - cd python && pytest test
```
Comment (setu4993) [inline recipe/meta.yaml:34]: Nice!

**Review by setu4993 — APPROVED** [review]

**Comment by h-vetinari** [comment]
> Tagging @h-vetinari in case they wanted to take a look at this independently again.

Sorry I didn't have time to look sooner. Looks good, thanks! :)

**Pull request merged**